.github/workflows/test.yml

```diff
@@ -62,14 +62,15 @@ jobs:
         uses: codecov/codecov-action@v1.5.2
   
   e2e_test:
+    needs: should_run_go_test
+    if: ${{ needs.should_run_test.outputs.should_skip != 'true' }}
     runs-on: ubuntu-latest
     timeout-minutes: 25
     steps:
       - uses: actions/setup-go@v2.2.0
         with:
           go-version: 1.18
       - uses: actions/checkout@v2
-      - uses: technote-space/get-diff-action@v6.0.1
```

Comment:
I don't follow why were removing this

Reply:
I added [this line](https://github.com/osmosis-labs/osmosis/blob/afef950bb23a304fe0fe76aa7fd5063d26e60798/.github/workflows/test.yml#L66) because I thought it does the same as the removed `git-diff`. On another look now, they do different things so I will restore it.

Thanks for catching that